includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -82,6 +83,12 @@ class UPE_Payment_Gateway extends WC_Payment_Gateway_WCPay {
 	 * @var string
 	 */
 	protected $checkout_title;
+	/**
+	 * WC_Payments_Localization_Service instance.
+	 *
+	 * @var WC_Payments_Localization_Service
+	 */
+	protected $localization_service;
```

Comment:
Isn't it inherited?

Reply:
Yes of course changed it https://github.com/Automattic/woocommerce-payments/pull/6920/commits/3dc1eb4d18404c33265e1217b9157f87c679920e thanks!